README_CELL-DINO_AND_CHANNEL-DINO.md

```diff
@@ -0,0 +1,29 @@
+The contents of this repo, including the code and model weights, are intended for research use only. It is not for use in medical procedures, including any diagnostics, treatment, or curative applications. Do not use this model for any clinical purpose or as a substitute for professional medical judgement.
+
+
+# Scaling Channel-Adaptive Self-Supervised Learning (CHANNEL-DINO)
+
+ [[`Paper `](https://openreview.net/forum?id=pT8sgtRVAf))] [[`BibTeX`](#citing-channeladaptivedino-and-dinov2)]
+
+Alice V. De Lorenci, Seungeun Yi, Théo Moutakanni, Piotr Bojanowski, Camille Couprie, Juan C. Caicedo, Wolfgang M. Pernice,
+
+with special thanks to Elouan Gardes for his contributions to the codebase.
+
+ [README](https://github.com/facebookresearch/dinov2/blob/main/docs/README_CHANNEL_ADAPTIVE_DINO.md)
+
+
+
+# Cell-DINO: Unbiased Image-based Embeddings for Cell Fluorescent Microscopy (CELL-DINO)
```

Comment:
Can we change this for the new paper title, please?
"Cell-DINO: Self-Supervised Image-based Embeddings for Cell Fluorescent Microscopy"